toolkit/components/downloads/moz.build

```diff
@@ -60,7 +60,7 @@ if CONFIG['OS_ARCH'] == 'WINNT':
 
 # XXX - Until Suite builds off XULRunner we can't guarantee our implementation
 # of nsIDownloadManagerUI overrides toolkit's.
-if not CONFIG['MOZ_SUITE']:
```

Comment:
I've tried this patch, but it doesn't seem to work. Looks like xulrunner/confvars.sh are ignored

> diff --git a/toolkit/components/downloads/moz.build b/toolkit/components/downloads/moz.build
> index 5f8ccbe..8019876 100644
> --- a/toolkit/components/downloads/moz.build
> +++ b/toolkit/components/downloads/moz.build
> @@ -45,7 +45,7 @@ if CONFIG['OS_ARCH'] == 'WINNT':
> 
> -if not CONFIG['MOZ_SUITE']:
> +if not CONFIG['MOZ_SUITE'] and not CONFIG['MOZ_EMBEDLITE']:
>      EXTRA_COMPONENTS += [
>          'nsDownloadManagerUI.js',
>          'nsDownloadManagerUI.manifest',
> diff --git a/xulrunner/confvars.sh b/xulrunner/confvars.sh
> index 4b2131d..9586545 100755
> --- a/xulrunner/confvars.sh
> +++ b/xulrunner/confvars.sh
> @@ -20,3 +20,4 @@ MOZ_USE_NATIVE_POPUP_WINDOWS=1
>  MOZ_PLACES=
>  MOZ_SERVICES_HEALTHREPORT=1
>  MOZ_DISABLE_EXPORT_JS=1
> +MOZ_EMBEDLITE=1